### DISCUSSION TITLE
Messages don't delivered both sided

### DISCUSSION THREAD

**Discussion by Dolise** [devlikeapro/waha#637] [Q&A]
I've started to face an issue that after session running a few days it's randomly start to not receive messages on a client level. In clients it's looks like this
<img width="776" alt="image" src="https://github.com/user-attachments/assets/c9fa7f2a-a930-4980-9e4b-4ba073f0ab09">

same when trying to send something.

Fixing by restart session

**Comment by devlikepro** [comment]
Hi! What engine do you use? Browser based WEBJS or NOWEB?
Could you the next time check the logs and attach it? Right after you stopped receiving messages
You can send it to waha@devlike.pro (it can have private info)

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
and the status of the session was `WORKING` anyway,right?
Do you use the latest version?

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
